pytradfri/util.py

```diff
@@ -13,14 +13,14 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-def load_json(filename: str) -> list | dict:
+def load_json(filename: str) -> list[Any] | dict[Any, Any]:
     """Load JSON data from a file and return as dict or list.
 
     Defaults to returning empty dict if file is not found.
     """
     try:
         with open(filename, encoding="utf-8") as fdesc:
-            return cast(Union[dict, list], json.loads(fdesc.read()))
+            return cast(Union[dict[Any, Any], list[Any]], json.loads(fdesc.read()))
```

Comment:
```suggestion
            return cast(Union[Dict[Any, Any], List[Any]], json.loads(fdesc.read()))
```

Reply:
What is the reason for lowercase `dict` not working here?

Reply:
The future annotations doesn't work within `cast`.

Reply:
Thanks!